rendering_resource_manager_service/service/settings.py

```diff
@@ -176,6 +176,7 @@
 # Slurm credentials (To be modified by deployment process)
 SLURM_USERNAME = 'TO_BE_MODIFIED'
 SLURM_KEY = 'TO_BE_MODIFIED'
+SLURM_PROJECT = 'TO_BE_MODIFIED'
```

Comment:
Should we set these to UNDEFINED, and then don't pass the option to slurm. For example, not all sites require a project.